backend/service/status/status_grpc.go

```diff
@@ -24,7 +24,7 @@ func (s *Service) Server(_ context.Context, _ *pb.EmptyRequest) (*pb.ServerStatu
 		Status: &pb.ServerStatus{
 			Name:       s.server.Name(),
 			Hostname:   s.server.Hostname(),
-			ListenURLs: s.server.ListenURLs(),
+			ListenUrls: s.server.ListenURLs(),
```

Comment:
I am surprised that go fmt does not complain about Url vs URL

Reply:
because it's `CODE GENERATED`